pydy/viz/scene.py

```diff
@@ -412,9 +426,10 @@ def create_static_html(self, overwrite=False, silent=False):
         simulation_outfile.write(json.dumps(self._simulation_data_dict, indent=4,
                                 separators=(',', ': ')))
         simulation_outfile.close()
-        if not silent: print("To view the visualization, open {}".format(
-                            os.path.join(dst, 'index.html')) +
-                            " in a WebGL compliant browser.")
+        if not silent:
+            print("To view the visualization, open {}".format(
```

Comment:
This no longer works. The user must view the html file using a server, right? If so we need to have this message tell how to do that.